### PR TITLE
This PR fixes issue #267: the KeyHandler Demo does not accept numpad input

### PR DESCRIPTION
**Pull request by fbsolo**
This PR fixes issue #267: the KeyHandler Demo does not accept numpad input

### CODE REVIEW

File: closure/goog/demos/keyhandler.html [hunk 1]
```diff
@@ -98,6 +98,20 @@
   }
 
   switch (e.keyCode) {
+
+  // Add the keycodes for the number-pad keys
+
+	case goog.events.KeyCodes.NUM_ZERO:
```
Comment (concavelenz) [inline closure/goog/demos/keyhandler.html]: Looks like perhaps you have some tabs in this file?

File: closure/goog/editor/plugins/removeformatting.js [hunk 2]
```diff
@@ -651,8 +651,21 @@ goog.editor.plugins.RemoveFormatting.prototype.removeFormattingWorker_ =
           continue;
 
         case goog.dom.TagName.P:
-          goog.editor.plugins.RemoveFormatting.appendNewline_(sb);
-          goog.editor.plugins.RemoveFormatting.appendNewline_(sb);
+
+		  // Make sure the "P" paragraph tag is not the first
```
Comment (concavelenz) [inline closure/goog/editor/plugins/removeformatting.js]: how is this change related to  keypad input?

File: closure/goog/demos/keyhandler.html (hunk 1 above)
Comment (fbsolo) [inline closure/goog/demos/keyhandler.html]: I apologize. Yes - my mistakes. I would fix these myself but I don't want to cause any problems. If you / whomever could fix this, I would be grateful.

Thank you!

File: closure/goog/editor/plugins/removeformatting.js (hunk 2 above)
Comment (fbsolo) [inline closure/goog/editor/plugins/removeformatting.js]: I apologize for this - it is not. I logged a number of separate pull requests and the change to this file got caught in this PR - PR #472.

For PR #472, only file

closure/goog/demos/keyhandler.html

should change.

Comment (joeltine) [inline closure/goog/editor/plugins/removeformatting.js]: Can you remove it from this PR?

File: closure/goog/demos/keyhandler.html (hunk 1 above)
Comment (joeltine) [inline closure/goog/demos/keyhandler.html]: It'd be easier if you just edit it. Just indent the cases properly with spaces and not tabs.

**Comment by concavelenz** [comment]
If nothing else, you can create a new branch based on master and apply just the changes you mean here. We can't accept this PR as is. Feel free to create a new PR when ready or reopen this when ready.
